bin/cli.js

```diff
@@ -14,7 +14,13 @@ function getParentBranch() {
 }
 
 function bump(type) {
-    return execSync(`npm version ${type}`);
+    const packageName = execSync(`cut -d "=" -f 2 <<< $(npm run env | grep "npm_package_name")`, {encoding: 'utf8'}).trim();
```

Comment:
You can do `npm info . name` to get the name of the project in the current folder, I don't know if it's ok for your use case

In the same idea (if it works), you could probably remove this line and do `npm show . versions -json`

Reply:
Nice, I looked a while for this. I'll change it.